src/pages/resultsView/fusion/Fusions.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ *
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+import * as React from 'react';
+import { observer } from "mobx-react";
+import { ResultsViewPageStore } from '../ResultsViewPageStore';
+import { observable } from 'mobx';
+import { MSKTab, MSKTabs } from '../../../shared/components/MSKTabs/MSKTabs';
+import Loader from '../../../shared/components/loadingIndicator/LoadingIndicator';
```

Comment:
I think we call it LoadingIndicator elsewhere so lets keep that convention so people know it's the the same thing